src/main/java/uk/ac/cam/cl/dtg/isaac/api/managers/EventNotificationEmailManager.java

```diff
@@ -171,7 +171,7 @@ public void sendFeedbackEmails() {
     ZonedDateTime sixtyDaysAgo = now.plusDays(EMAIL_EVENT_FEEDBACK_DAYS_AGO);
```

Comment:
So is this. Variables named after the value of the constant being assigned to them. This is horrid.